docs/coding-style.md

```diff
@@ -40,6 +50,10 @@ Local functions should always be placed at the end of a method.
     }
 ```
 
+### Separation
+
+Individual members must be separated by empty line, except sequence of constants, fields and abstract members. These should not be separated by empty lines.
```

Comment:
This change is out of scope of the original thing that was discussed on the card.

We use this also for block of single-line properties, as those are the same as fields. And adding empty lines here makes it harder to read - to see boundaries of property-region and to understand what the "state" of the object is.

Reply:
Examples:

Easy to read:
https://github.com/SonarSource/sonar-dotnet/blob/01bfee73f97a72e18093db7acb876bd985dcc104/analyzers/src/SonarAnalyzer.Common/AnalysisContext/SonarCompilationStartAnalysisContext.cs

Hard to read - hard to find that it changed from a sequence of properties to constructors or methods:
https://github.com/SonarSource/sonar-scanner-msbuild/blob/master/src/SonarScanner.MSBuild.Shim/ProjectInfoAnalysisResult.cs

or

https://github.com/SonarSource/sonar-dotnet/blob/master/analyzers/src/SonarAnalyzer.CFG/Sonar/Blocks/Block.cs

Reply:
>This change is out of scope of the original thing that was discussed on the card.

there's no content change, it's "Clean as you code" by organizing the same information

Reply:
I won't open a discussion with the bubble for adding a `### Separation` and moving content from one place to another in the coding style

I really don't understand your comment

Reply:
Indeed, I overlooked the red line in github UI 😒

Reply:
I don't mind the `### Separation` section.